src/assets/js/protocol/protocol-sidemenu.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    'use strict';
+
+    var $doc = document.documentElement;
+    var menu = document.querySelector('.mzp-c-sidemenu');
+    var menuToggle = document.querySelector('.mzp-js-toggle');
+
+    // Add class to reflect javascript availability for CSS
```

Comment:
Should we remove this from the sidemenu JS now that we have a separate base JS file?